IDEOorg/steps
# #161304887 Show Client Platfrom

#### What does this PR do?
Show Client Platform
#### Description of Task to be completed?
- modify UserLayout.tsx file
- modify the Profile.tsx component file to display the platform
#### How should this be manually tested?
- Clone the repository from `https://github.com/IDEOorg/steps.git` and `cd steps`
- Run the application with `npm start`
- Navigate to your browser with the URL `http://localhost:3000`
- Login with your details if registered
- Click on my clients and click on any of the clients
- Check below you should see the platform for that client at the bottom of the side panel

#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#161304887](https://www.pivotaltracker.com/story/show/161304887)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-24 at 4 00 04 pm" src="https://user-images.githubusercontent.com/5827585/47440731-aa822900-d7a6-11e8-94f4-de8261000c72.png">


#### Questions:
- N/A

[Delivers #161304887]
